tests/misc/unzip.ae.expected

```diff
@@ -1,2 +1,2 @@
 
-unknown
+unsat
```

Comment:
This change is normal. I changed the test in order to use the same one for both native and SMT language.